docs/real-world-dsp/low-level-debugging.md

```diff
@@ -9,8 +9,9 @@ description: >-
 
 ## Breakpoint and watch
 
-The first and maybe most instinctive way to check if a code is working as expected is to put a breakpoint at a critical line of code. In that way it is possible to check if the micro-controller is going through a certain instruction and do a step-by-step execution of the code starting from the breakpoint.
+The first and maybe most instinctive way to check if a code is working as expected is to put a breakpoint at a critical line of code. In that way it is possible to check if the micro-controller is going through a certain instruction and to do a step-by-step execution of the code starting from the breakpoint.
 
+<!-- what does "the view focuses" below mean? -->
```

Comment:
I did not understand "the view focuses"

Reply:
the view "moves to show the breakpoint" :)